### ISSUE TITLE
Can you divide the 400 epoch learning by 200 and continue it?

### ISSUE THREAD

**Issue by leeyunhome** [ultralytics/yolov5#2441]
## ❔Question

Hello,

By specifying the number of epochs as 200, the trained model is used as the pretrained weight.
If you train 200 epochs again

Is it the same as spinning a total of 400 epochs at once?

Thank you.

**Comment by glenn-jocher** [comment]
@leeyunhome yes you can do both, but they are **not** the same!

Training starts a warmup period and LR scheduler that ramps from lr0 to near zero, so training twice will run the warmup and scheduler cycle twice. You can see the LR in your logging plots:
<img width="1283" alt="Screen Shot 2021-03-12 at 9 05 09 AM" src="https://user-images.githubusercontent.com/26833433/110973431-1c288680-8312-11eb-825d-406bd07afc7f.png">
